### ISSUE TITLE
Revive the Triage Team

### ISSUE THREAD

**Issue by ctcpip** [expressjs/discussions#227]
topic from agenda overflow from meeting today.  to someone with the requisite permissions: please add the priority label so the bot picks it up for the next meeting

**Comment by jonchurch** [comment]
This is something I want to talk about for sure. 

I am marginally involved in managing Triagers, I have stopped adding people bc I just don't feel I/we are equipped to give them a good experience rn. 

But we should def get to that place ASAP

**Comment by ctcpip** [comment]
@wesleytodd I think it needs the `top priority` label for the bot (?)

**Comment by UlisesGascon** [comment]
Thanks for picking it up, @ctcpip! I was just about to create an issue.

Okay, here are my two cents on the topic:

The current definition of the role is somewhat defined ([see](https://github.com/expressjs/express/blob/master/Triager-Guide.md)) and the steps to become a triage member [here](https://github.com/expressjs/express/blob/master/Contributing.md#becoming-a-triager).

From what I see, this is a great role to get involved in the project, and I love this idea. However, currently, the situation is not very good in terms of triage membership experience. Also, in many cases, the implication is very low, and the retention is also low. In my opinion, this is because the support and role definition are not yet there.

I think that triage members can easily become collaborators and make meaningful contributions (I started as triage myself in this project), but we need to provide good mentorship.

**Here are some ideas to change the dynamics:**

- Define a Captain to become the mentor of the group (I volunteer myself if needed) so that person can delegate tasks and spot easy contributions in the project to the team members so they can learn and add value from a safe environment.
- Change the nomination process by getting nominated for the role rather than self-nomination. Anyone in the org can nominate new triage team members. The idea is that people who start helping with moderation and replying to issues or commenting/reviewing PRs can get nominated.
- Approve and onboard the cases that have positive feedback from org members like [this](https://github.com/expressjs/express/issues/5463) and [this](https://github.com/expressjs/express/issues/5487).
- Close all pending self-nominations and respond to the issues with the new process in place [here](https://github.com/expressjs/express/issues?q=is%3Aissue+is%3Aopen+triage).
- Purge the team members that are not active; currently, there are 37 members ([see](https://github.com/orgs/expressjs/teams/triagers)), most of them inactive for a long time.
- Improve the definitions of what is `inactive` and how to review it programmatically, so we can auto-purge.


Overall, let's empower this team to help us manage the huge amount of issues and PRs across the long list of repositories that we have and to build the next generation of collaborators 🥳

**Comment by wesleytodd** [comment]
I swapped to two labels for the bot. The Working session uses `top priority` and the TC meeting uses `tc agenda`. Not sure if folks like that, but feels alright to me.

> Define a Captain to become the mentor of the group (I volunteer myself if needed)

Love this!!

> Change the nomination process by getting nominated for the role rather than self-nomination.

So the recommended process would be to help out in issues *first* then be nominated? And so we need docs telling people to just *start* helping?  I like it, but just making sure I am clear.

And yeah I am all in support of the rest of your suggestions I think! This is great, really glad to see this happening!

**Comment by UlisesGascon** [comment]
> So the recommended process would be to help out in issues first then be nominated? And so we need docs telling people to just start helping? I like it, but just making sure I am clear.

Yes!

> And yeah I am all in support of the rest of your suggestions I think! This is great, really glad to see this happening!

Cool! Let's close the details on Monday as we have the TC meeting  :)

**Comment by UlisesGascon** [comment]
This was discussed on the last TC meeting, and the plan is approved. [recording (relevant from 11:30)](https://www.youtube.com/live/fqqJP2S-NHk?si=wbOU0J0Ta6HX3Pev&t=683) :partying_face:

**Comment by UlisesGascon** [comment]
### Next steps:
- [ ] Resolve: https://github.com/expressjs/express/issues/5463
- [ ] Resolve: https://github.com/expressjs/express/issues/5487
 - [ ] Change the nomination process by getting nominated for the role rather than self-nomination
 - [ ] Close all pending self-nominations and respond to the issues with the new process in place [here](https://github.com/expressjs/express/issues?q=is%3Aissue+is%3Aopen+triage).
 - [ ] Purge the team members that are not active (1w confirmation). 
 - [ ] Improve the definitions of what is `inactive` and how to review it programmatically, so we can auto-purge.

**Comment by UlisesGascon** [comment]
Hi friend! You are reading this message because you are currently a triage team member for our organizations ([expressjs](https://github.com/expressjs), [pillarjs](https://github.com/pillarjs), and [jshttp](https://github.com/jshttp)). Currently, we are reorganizing the team, and we want to confirm if you are interested in participating in this new phase ([details](https://github.com/expressjs/discussions/issues/227#issuecomment-2048566518)).

If you want to remain active in this role and participate in the new initiative, please reply to this thread in GitHub, so we can keep your access and send you the additional information :+1:

Current team members that **require confirmation** (marked means confirmed) **before Monday 22nd**:
- [ ] @0ss
- [ ] @3imed-jaberi
- [ ] @AuggieH
- [ ] @G-Rath
- [ ] @MohammadXroid
- [ ] @NawafSwe
- [ ] @NotMoni
- [ ] @Sushmeet
- [ ] @VigneshMurugan
- [ ] @aravindvnair99
- [ ] @dakshkhetan
- [ ] @davidmashe
- [ ] @digitaIfabric
- [ ] @e-l-i-s-e
- [ ] @enyoghasim
- [ ] @fed135
- [ ] @firmanJS
- [ ] @getspooky
- [ ] @ghinks
- [ ] @ghousemohamed
- [ ] @gireeshpunathil
- [ ] @import-brain
- [ ] @jake32321
- [ ] @lekanikotun
- [ ] @lucasraziel
- [ ] @marsonya
- [ ] @mastermatt
- [ ] @maxakuru
- [ ] @mlrawlings
- [ ] @rodion-arr
- [ ] @tarunyadav1
- [ ] @tunniclm
- [ ] @yg


Note: this list does not include new additions (from today), Captains or TC members.

**Comment by 3imed-jaberi** [comment]
Hello @UlisesGascon, 
I confirm, and if possible could you please add me to the other teams too, I can participate by code contribution/maintenance too.

**Comment by aravindvnair99** [comment]
@UlisesGascon Yes, I've been active and will be continuing.

**Comment by dakshkhetan** [comment]
@UlisesGascon Yes, I am eager to remain active in this role and participate in the new initiative. Thanks!

**Comment by gireeshpunathil** [comment]
> Hi friend! You are reading this message because you are currently a triage team member for our ...

TLDR; I have changed roles and this project is not in my priority list anymore, so pls remove my name.

I had conflicts with @dougwilson (for example https://github.com/expressjs/discussions/issues/100#issuecomment-664092436) in the past which neither of us took steps to resolve, and I stopped being active.

great to see the project reviving again with lot of new faces and new initiatives; one of my dream is to see express 5 released and used.

**Comment by lucasraziel** [comment]
@UlisesGascon I would like to participate in this new phase.

**Comment by Sushmeet** [comment]
@UlisesGascon This is exciting. I learnt a lot in my first stint and found the community very supportive and kind. I would like to be included and become active again.

**Comment by enyoghasim** [comment]
@UlisesGascon i know i have been inactive for a while, but i love to be included and become active with contributions again

**Comment by rodion-arr** [comment]
Hi, I can't invest time at this stage, thank you all 🤝

**Comment by G-Rath** [comment]
While I'd love to be involved, sadly I've not got the bandwidth - I assume the offer will remain though for future so will let folks know if I get the bandwidth in future :)

**Comment by UlisesGascon** [comment]
Thanks for all the responses! :heart: 

I just updated the team to reflect the correct members based on the responses, please direct ping me in case of any issue.

I made all the changes needed in the repos, documentation, etc.. so I close this issue.

**Next steps for the @expressjs/triagers**
- Follow up with our calendar events and join our Slack. [details](https://openjsf.org/collaboration)
- Please join the channels `#express` and `#express-triage-team`
